libs/ledgerjs/packages/hw-app-exchange/src/Exchange.ts

```diff
@@ -30,9 +30,9 @@ const CHECK_PARTNER_COMMAND = 0x05;
 const PROCESS_TRANSACTION_RESPONSE = 0x06;
 const CHECK_TRANSACTION_SIGNATURE = 0x07;
 const CHECK_PAYOUT_ADDRESS = 0x08;
-const CHECK_ASSET_IN = 0x08;
 const CHECK_REFUND_ADDRESS = 0x09;
 const SIGN_COIN_TRANSACTION = 0x0a;
+const CHECK_ASSET_IN_AND_DISPLAY = 0x0b;
```

Comment:
I am not sure this is the code to use.
I didn't test with `CHECK_ASSET_IN_AND_DISPLAY = 0x0B`. Are you sure about the whole behaviour? I thought the app-exchange awaits for another APDU command in this flow.

Reply:
https://github.com/LedgerHQ/app-exchange/blob/8be8d7ee5100d432fae6c20ae929aa8c07e49537/src/apdu_parser.c#L198-L202
````
    if (instruction == CHECK_ASSET_IN_LEGACY_AND_DISPLAY &&
        (subcommand == SELL || subcommand == FUND)) {
        PRINTF("Replacing legacy value for CHECK_ASSET_IN_AND_DISPLAY instruction\n");
        instruction = CHECK_ASSET_IN_AND_DISPLAY;
    }